### ISSUE TITLE
Report whether ribbon is shown or hidden when CONTROL+F1 is pressed 

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#11945]
### Is your feature request related to a problem? Please describe.

In Windows 10 applications such as WordPad and File Explorer, as well as in Microsoft Office, the ribbon can be shown or hidden with CONTROL+F1.

While this is mostly a visual option, it does alter the behaviour if the user then presses alt to access the ribbon and the arrows, tab or control+tab to move through options.

- With the ribbon shown, alt, then tab moves through the items in the current ribbon tab.
- With the ribbon hidden, alt, then tab moves from the ribbon header to the quick access toolbar (you need to press alt, then arrow to a ribbon header, then enter and THEN you can tab through the options in that ribbon.

### Describe the solution you'd like

Ideally, NVDA should report "Ribbon shown" or "Ribbon hidden" or something similar to designate the state of the ribbon when control+f1 is pressed.  This is both because it changes the behaviour, and also because there are very few keystrokes which should not elicit a response from NVDA.

### Describe alternatives you've considered

Leaving it as it is, does not cause a large amount of confusion or problems, so it can be assigned a low priority, but I still feel it is worth addressing.

**Comment by Adriani90** [comment]
cc: @CyrilleB79

**Comment by CyrilleB79** [comment]
Using Microsoft® Word 2016 MSO (Version 2410 Build 16.0.18129.20158) 32 bits.
It seems fixed by an UIA notification from Word reporting the ribbon's new state.

@Qchristensen could you confirm this?

**Comment by Adriani90** [comment]
Yes this is fixed, testing with NVDA 2025.1 Beta. Closing as works for me. if you are still having this issue, please comment and we can reopen.
